webpack-test/cases/errors/case-sensitive/warnings.js

```diff
@@ -1,4 +1,4 @@
 module.exports = [
-	[/There are multiple modules with names that only differ in casing/, /case-sensitive.A\.js/, /case-sensitive.a\.js/],
-	[/There are multiple modules with names that only differ in casing/, /case-sensitive.B.file\.js/, /case-sensitive.b.file\.js/]
+	[/There are multiple modules with names that only differ in casing/, /(case-sensitive.A\.js)|(case-sensitive.B.file\.js)/, /(case-sensitive.a\.js)|(case-sensitive.b.file\.js)/],
+	[/There are multiple modules with names that only differ in casing/, /(case-sensitive.A\.js)|(case-sensitive.B.file\.js)/, /(case-sensitive.a\.js)|(case-sensitive.b.file\.js)/]
```

Comment:
Don't change test under `webpack-test`, try your best to pass it or track it in a issue